templates/introduction.html

```diff
@@ -12,31 +12,33 @@
 {% block main %}
 
     {% block intro_content %}
-      {% if content.block.primary_content %}
-          {% for content_block in content.block.primary_content %}
-              {% include 'partials/introduction/basic.html' %}
-          {% endfor %}
-      {% endif %}
-
-      {%- if legal_basis -%}
-          <h2 class="ons-u-mb-xs" data-qa="legal-response">{{ _("Your response is legally required") }}</h2>
-          <p class="ons-u-mb-m" data-qa="legal-basis">{{ legal_basis }}</p>
-      {%- endif -%}
-
-      {% block start_survey %}
-          {% include 'partials/introduction/start-survey.html' %}
-      {% endblock start_survey %}
-
-      {% if content.block.preview_content %}
-          {% set intro = content.block.preview_content %}
-          {% include 'partials/introduction/preview.html' %}
-      {% endif %}
-
-      {% if content.block.secondary_content %}
-          {% for content_block in content.block.secondary_content %}
-              {% include 'partials/introduction/basic.html' %}
-          {% endfor %}
-      {% endif %}
+        {% if content.block.primary_content %}
+            {% for content_block in content.block.primary_content %}
+                {% set titleTag = 'h1' %}
```

Comment:
Since we use snake case throughout these files I wouldn't start introducing camel case.

Reply:
Done